test/com/comeet/RoomsDaoTest.java

```diff
@@ -0,0 +1,114 @@
+package com.comeet;
+
+import java.io.IOException;
+import java.security.InvalidParameterException;
+import java.util.ArrayList;
+
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.comeet.exchange.ExchangeResourceException;
+import com.comeet.utilities.ApiLogger;
+
+import junit.framework.Assert;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RoomsDaoTest {
+    
+    
+    @InjectMocks 
+    RoomsDao dao;
+    
+    @Mock
+    ApiLogger log;
+    
+    @Before
+    public void setUp() throws IOException {
+        //ApiLogger.setup();
+    }
+    
+    //GetCriteria tests
+    //Note: This method calls a data repository class method therefore
```

Comment:
To make these tests work independently of our db, you should really be injecting a mock `sqlConnection` with known data.